src/translations/en-US.json

```diff
@@ -789,5 +789,40 @@
         "value": "Date is not in {format} format",
         "comment": "Placeholder '{format}' is used for showing date format. Example 'Date is not in MMMM/YY/DD format'. Don't translate placeholder '{format}'.",
         "limit": 0
+    },
+    "mvf.operator.all": {
+        "value": "all",
```

Comment:
Please provide an example. Translation can be dependent on the gender of the subject.